packages/hub/src/app/ai/api/system-prompt/route.ts

```diff
@@ -0,0 +1,5 @@
+import { squiggleSystemPrompt } from "@quri/squiggle-ai/server";
+
+export async function GET() {
+  return new Response(squiggleSystemPrompt);
```

Comment:
https://squigglehub.org/ai/api/system-prompt will be useful for checking that system prompt was generated correctly (many ways to make a mistake there, e.g. once I forgot an `await`, and documentation bundle was injected as `[object Promise]`)